cmd/identities/import.go

```diff
@@ -17,7 +17,17 @@ import (
 var ImportCmd = &cobra.Command{
 	Use:   "import <file.json [file-2.json [file-3.json] ...]>",
 	Short: "Import identities from files or STD_IN",
-	Example: `$ kratos identities import file.json
+	Example: `
+$ cat >./file.json <<EOF
+{
+    "schema_id": "default",
+    "traits": {
+        "email": "foo@ory.sh"
```

Comment:
```suggestion
        "email": "foo@example.com"
```
Maybe example.com is better, dont want to get spammed on ory.sh ;).

Reply:
Yeah sure changed it now!